src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { filterSlice } from '../../features/filter';
+import { Status } from '../../types/Status';
 
 export const TodoFilter: React.FC = () => {
+  const status = useAppSelector(state => state.filter.status);
+  const query = useAppSelector(state => state.filter.query);
+  const dispatch = useAppDispatch();
+
+  const handleChangeStatus = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    dispatch(filterSlice.actions.setStatus(e.target.value as Status));
```

Comment:
Ensure that the `setStatus` action is correctly defined in the `filterSlice`. If this action does not exist, it will cause an error when trying to dispatch it.